contribs/gmf/apps/mobile_alt/index.html.ejs

```diff
@@ -31,6 +31,14 @@
         gmf-displayquerywindow-featuresstyle="::mainCtrl.queryFeatureStyle"
         gmf-displayquerywindow-defaultcollapsed="false">
       </gmf-displayquerywindow>
+      <div
+        class="gmf-mobile-measure"
+        gmf-mobile-measurearea
+        gmf-mobile-measurearea-active="mainCtrl.measureAreaActive"
+        gmf-mobile-measurearea-precision="2"
```

Comment:
In 2.5 it's plane to remove all those kink of configuration to replace the by config object injection ...

Reply:
Okay, but what about now?  Should I change the config of this component and make it not the same as the others?  I think this should all be done in 2.5.

Thoughts?

Reply:
No problem for me :-)